src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -159,6 +160,11 @@ private PGPPublicKey loadPgpPublicKey(String encryptionPublicKey) {
             return pdfCreator.createFromTemplates(((LetterRequest) letter).documents);
         } else if (letter instanceof LetterWithPdfsRequest) {
             return pdfCreator.createFromBase64Pdfs(((LetterWithPdfsRequest) letter).documents);
+        } else if (letter instanceof LetterWithPdfsAndNumberOfCopiesRequest) {
+            return pdfCreator
+                .createFromBase64PdfWithCopies(
+                    ((LetterWithPdfsAndNumberOfCopiesRequest) letter).documents
+                );
```

Comment:
The approach we've used in orchestrator and payment-processor is probably more elegant, I'll look into it on a separate occasion.